### PR TITLE
Feat: Add cancel billing sessions, billing alerts, assume no plan is trial plan

### PR DESCRIPTION
**Pull request by Mikehrn**

### CODE REVIEW

**Review by andrewwallacespeckle — COMMENTED** [review]
The code looks good here, apart from the 1 comment I left.

I'm not sure how to best test this, will sync with you on this.

File: packages/frontend-2/components/settings/workspaces/Billing.vue [hunk 1]
```diff
@@ -38,14 +44,16 @@
                 {{ isTrialPeriod ? 'First payment due' : 'Next payment due' }}
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/settings/workspaces/Billing.vue:44]: image
Just checking that the pretitle `Current plan` should be font-medium, but the other pretitles are not.

Comment (Mikehrn) [reply packages/frontend-2/components/settings/workspaces/Billing.vue:44]: No, it wasn't supposed to be 😁

File: packages/frontend-2/components/settings/workspaces/Billing.vue [hunk 2]
```diff
@@ -175,39 +186,50 @@ const isPaidPlan = computed(
     currentPlan.value?.name !== WorkspacePlans.Unlimited
 )
 const isTrialPeriod = computed(
-  () => currentPlan.value?.status === WorkspacePlanStatuses.Trial
+  () =>
+    currentPlan.value?.status === WorkspacePlanStatuses.Trial ||
+    !!currentPlan.value?.status
```
[the remark this answers is not in the document]
Comment (Mikehrn) [reply packages/frontend-2/components/settings/workspaces/Billing.vue]: You're right, that was incorrect

**Review by andrewwallacespeckle — APPROVED** [review]

**Pull request merged**